internal/npm_install/test/golden/@angular/core/BUILD.bazel.golden

```diff
@@ -4,6 +4,8 @@ load("@build_bazel_rules_nodejs//internal/npm_install:node_module_library.bzl",
 filegroup(
     name = "core__files",
     srcs = [
+        "//:node_modules/@angular/core/ANGULAR_PACKAGE",
```

Comment:
this change suggests we don't really want a golden file for this package... how much of this delta do I care about? I don't think any of this is related to the fix